package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tag-release",
-  "version": "5.14.0",
+  "version": "5.14.2",
```

Comment:
Seems weird that it changes the version to what it should have been. We need to figure this out... so this doesn't continue to happen